lnwallet/interface.go

```diff
@@ -281,6 +281,9 @@ type BlockChainIO interface {
 	// GetBlock returns the block in the main chain identified by the given
 	// hash.
 	GetBlock(blockHash *chainhash.Hash) (*wire.MsgBlock, error)
+
+	// GetBlockHeader returns the header for the block with the given hash.
+	GetBlockHeader(blockHash *chainhash.Hash) (*wire.BlockHeader, error)
```

Comment:
After the latest round of offline discussions, I don't think we'll need this anymore? Since it's possible due to block time skew we start too early and miss a set of events for a channel.

Reply:
Yep, will remove and go with a basic first iteration as discussed.